RecoTracker/FinalTrackSelectors/src/TrackMVAClassifierBase.cc

```diff
@@ -41,18 +43,19 @@ TrackMVAClassifierBase::TrackMVAClassifierBase( const edm::ParameterSet & cfg )
 
 }
 
+
+#include "DataFormats/VertexReco/interface/Vertex.h"
```

Comment:
#include is a preprocessing directive, and therefore compilation time shouldn't depend on where it is placed.
I think it can be a good practice to put all them together at the beginning of the cpp file, so that they are all listed together and not scattered around the file, which makes easier checking whether something is missing or placed at a wrong order. At this point it is mostly a matter of style, I admit, but it can help when the complexity of the code increases.